### DISCUSSION TITLE
b站页面改版

### DISCUSSION THREAD

**Discussion by Winston-62** [the1812/Bilibili-Evolved#2483] [综合讨论 / General]
刚刚b站推送了测试版的新网页排版，有很多功能变得很奇怪...
![屏幕截图 2021-11-04 192047](https://user-images.githubusercontent.com/64537216/140305121-f0c60b6e-f557-4c8d-9597-72afa5c4a46b.png)
.

**Comment by the1812** [comment]
新版首页应该适配过的: #2209

**Reply by Winston-62** [reply]
1.![image](https://user-images.githubusercontent.com/64537216/140307983-201cd9d3-b9b1-4833-a158-6a66deec71f9.png)
排行分区打开后上面会有一块白色横幅
2.![image](https://user-images.githubusercontent.com/64537216/140308090-3539dcb2-b21b-4042-af85-94609272f9ef.png)
左侧的  主站  下面的那三个图标显示有些问题
3.![image](https://user-images.githubusercontent.com/64537216/140308559-ccbfa974-0c79-4c33-87ca-d401d229b47b.png)
有部分新的栏目没有适配  夜间模式
4.![image](https://user-images.githubusercontent.com/64537216/140308817-35f505a1-5659-49a7-a61f-28397ff23442.png)
希望能在使用   首页过滤  功能后可以把相应过滤掉的分区隐藏起来，毕竟分区真的有点太多啦，有很多不用的会不小点到。
感谢，Y.Y

**Reply by Winston-62** [reply]
![image](https://user-images.githubusercontent.com/64537216/140310334-081c3612-f6e9-415d-960b-6164570c095f.png)
鬼畜区也有些问题。

**Reply by Geeyun-JY3** [reply]
3. 新话题刚上还没适配，和新版首页无关，任意相关功能都是：https://github.com/the1812/Bilibili-Evolved/discussions/1450#discussioncomment-1583312, https://github.com/the1812/Bilibili-Evolved/issues/2136#issuecomment-959647600

**Reply by Winston-62** [reply]
好的，感谢
